third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+**Trigger** - The **user** triggers this use-case upon demand.
+
+**Successful Result** - The **user** receives *image* messages at the desired frame rates.
+
+**Main Scenario**
+1. The **user** sends a *start streaming* message with the requested profiles listed.
+2. The **server** starts publishing *image* messages for each requested profile.
+
+**Variations**
+
+None.
+
+**Exceptions**
+1. The device is already streaming data from this sensor
+    1. The **server** will decline the request //TODO - should handle throw in rs-dds-server?
+    2. Terminate the use-case. //TODO - send failure *notification*? Currently not in the ICD
```

Comment:
WIP?

Reply:
Yes, still WIP.
Current implementation doesn't notify the user on such failure, the user can detect failure based on active streams. While writing the use case I thought it might be more user friendly to send a failure notification, but it is not implemented yet and I'm also waiting for Embedded team comments about it.

Reply:
Please rephrase the "2." so that it will be explicit that this is WIP

Reply:
Added

Reply:
ok